.github/workflows/ci.yml

```diff
@@ -80,6 +80,8 @@ jobs:
     env:
       CLCACHE_DIR: ${{ GITHUB.WORKSPACE }}\clcache
     steps:
+      - name: Support longpaths
+        run: git config --system core.longpaths true
```

Comment:
Did you run into issues? What were they? If it's an easy change, I would prefer avoiding long paths, as there may be tools that don't support them.

Reply:
Yes, unfortunately cloning the submodules of the AWS SDK failed on Windows due to long filenames (https://github.com/lordgamez/nifi-minifi-cpp/runs/3136241080?check_suite_focus=true):

```
2021-07-22T17:09:54.3295498Z          fatal: cannot write keep file 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/.git/modules/aws-common-runtime/aws-crt-cpp/modules/aws-common-runtime/s2n/modules/tests/cbmc/aws-verification-model-for-libcrypto/objects/pack/pack-4ed6079c9d3778b360b478480245f806aae00bf8.keep': Filename too long
2021-07-22T17:09:54.3297474Z          fatal: fetch-pack: invalid index-pack output
2021-07-22T17:09:54.3299454Z          fatal: clone of 'https://github.com/awslabs/aws-verification-model-for-libcrypto.git' into submodule path 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/crt/aws-crt-cpp/crt/s2n/tests/cbmc/aws-verification-model-for-libcrypto' failed
2021-07-22T17:09:54.3301603Z          Failed to clone 'tests/cbmc/aws-verification-model-for-libcrypto'. Retry scheduled
2021-07-22T17:09:54.3302887Z          Cloning into 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/crt/aws-crt-cpp/crt/s2n/tests/cbmc/templates'...
2021-07-22T17:09:54.3304157Z          Cloning into 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/crt/aws-crt-cpp/crt/s2n/tests/litani'...
2021-07-22T17:09:54.3306108Z          Cloning into 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/crt/aws-crt-cpp/crt/s2n/tests/cbmc/aws-verification-model-for-libcrypto'...
2021-07-22T17:09:54.3309030Z          fatal: cannot write keep file 'D:/a/nifi-minifi-cpp/nifi-minifi-cpp/build/thirdparty/aws-sdk-cpp-src/.git/modules/aws-common-runtime/aws-crt-cpp/modules/aws-common-runtime/s2n/modules/tests/cbmc/aws-verification-model-for-libcrypto/objects/pack/pack-4ed6079c9d3778b360b478480245f806aae00bf8.keep': Filename too long
```